### ISSUE TITLE
String-to-float conversion failure (usually caused by NULL values)

### ISSUE THREAD

**Issue by mikegough** [consbio/mpilot#11]
When the input csv contains NULL values in any of the input fields, it generates the error message below. This is a very common problem in EEMS, but the traceback does not effectively communicate what is causing it. 

Could we trap this error and let the user know A) which field is causing the problem, and B) what the likely reason for it is? 

Here is some suggested text for the error message:

_There was an error converting one of the values in the field [FIELD NAME] to a floating point number.  Please check to make sure there aren't any NULL values in this field, and that all the values are numeric._ 

```
Traceback (most recent call last):
  File "C:\Python27\ArcGIS10.6\lib\site-packages\mpilot\commands.py", line 140, in run
    {arg.name: arg.value for arg in self.arguments}
  File "C:\Python27\ArcGIS10.6\lib\site-packages\mpilot\libraries\eems\csv\io.py", line 53, in execute
    values.append(float(row[idx]))
ValueError: could not convert string to float: 
```

**Comment by nikmolnar** [comment]
Yes, that'd be an easy fix.

**Comment by nikmolnar** [comment]
This is implemented in version `1.0.0b21`. The error looks like this:

```
Problem: The data file contains an invalid value in the field b on line 3.
Solution: Verify that the data file doesn't contain any empty or NULL values, and that all values are numeric.
```
